src/test-utils/dom/cards/index.ts

```diff
@@ -86,6 +86,10 @@ export default class CardsWrapper extends ComponentWrapper {
     return this.findComponent(`.${tableStyles['tools-pagination']}`, PaginationWrapper);
   }
 
+  findFooterPagination(): PaginationWrapper | null {
+    return this.findComponent(`.${styles['footer-pagination']}`, PaginationWrapper);
+  }
```

Comment:
What is the point of making this util public?

Reply:
I was also not sure about its utility, but your comment is just another datapoint to confirm it. I removed it.